boards/common/nrf51/include/cfg_rtt_default.h

```diff
@@ -29,11 +29,21 @@ extern "C" {
  * @name    Real time counter configuration
  * @{
  */
-#define RTT_DEV             (1)             /* NRF_RTC1 */
-#define RTT_MAX_VALUE       (0x00ffffff)
-#define RTT_FREQUENCY       (1024)
+#ifndef RTT_DEV
+#define RTT_DEV             (1)                 /* NRF_RTC1 */
+#endif
+
+#define RTT_MAX_VALUE       (0x00ffffff)         /* 24bit */
+#define RTT_MAX_FREQUENCY   (32768U)             /* in Hz */
```

Comment:
RTT_MAX_FREQUENCY and RTT_MIN_FREQUENCY constants are not used by the nrf driver. So I either remove them or add some pre-processor logic that checks RTT_FREQUENCY is within bounds.

Reply:
I was planning on adding that in `periph/rtt.h` once all arch define these variables (#13874 ). For now they are just documenting. I could add that logic in the `.c` file if you want

Reply:
> I could add that logic in the .c file if you want

I think that would make sense to fail early if the value is invalid. I won't insist.